src/main/java/baritone/Baritone.java

```diff
@@ -1,219 +1,220 @@
-/*

```

Comment:
I don't even know what to say about this, what could you possibly do to remove and re-add the whole file.